contracts/core/OriginCore.sol

```diff
@@ -412,4 +474,120 @@ contract OriginCore is OriginCoreInterface, OriginCoreConfig {
 
         return kernelHash;
     }
+    /**
+     * @notice private method to check vote validity.
+     *
+     * @dev This method will revert the transaction if validator is not
+     *       eligible for voting.
+     *
+     * @param _voteHash The hash of the transition part of the meta-block
+     *                    header at the source block.
```

Comment:
Actually it is the hash of the vote object. Maybe a copy/paste error? Or I don't get it 🤔